SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.x.x   | :white_check_mark: |
+| < 4.0.0 | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We believe our user community and unaffiliated security researchers play an important role in
+helping to keep Fauna and our customers secure. If you think you have found a security issue,
+please inform us by sending an email to security@fauna.com.
+
+Learn more about reporting a vulnerability at [support.fauna.com](https://support.fauna.com/hc/en-us/articles/4449808041115-How-do-I-report-a-vulnerability).
```

Comment:
Can we point folks directly to trust.fauna.com? Support will not accept security reports, and contacting support about a report can disqualify the bounty. The article in support is kind of a last attempt to get folks not to create a support ticket. I am afraid that pointing to support.fauna.com at all encourage submitting a ticket while in the helpdesk.